src/RawPOSIX/src/safeposix/syscalls/sys_calls.rs

```diff
@@ -131,67 +131,27 @@ impl Cage {
     }
 
     /*
-    *   exec() will only return if error happens 
+    *   Here is the linux man page for execve: https://man7.org/linux/man-pages/man2/execve.2.html
+    *   exec() will only return if error happens
+    *   Unlike the `exec` syscalls from linux manual, exec syscall here does not take any arguments,
```

Comment:
This comment section starting from here is a bit hard to read. Can we clean it up a bit?